src/markdown/footnotes.scss

```
@@ -0,0 +1,26 @@
+// stylelint-disable selector-max-type
+// stylelint-disable selector-no-qualifying-type
+.footnotes {
```

One question I have here @talum Does `.footnotes` only occur in `markdown-body`? If so I'd like to nest this.

```suggestion
.markdown-body .footnotes {
```

---

Ah! Yes, great point. It does only ever occur within `.markdown-body`.

Just to confirm this looks fine, I made the change locally in dotcom and tested it. ✅